roles/openshift_health_checker/openshift_checks/logging/curator.py

```diff
@@ -9,46 +9,33 @@ class Curator(LoggingCheck):
     name = "curator"
     tags = ["health", "logging"]
 
-    logging_namespace = None
-
     def run(self):
-        self.logging_namespace = self.get_var("openshift_logging_namespace", default="logging")
-        curator_pods, error = super(Curator, self).get_pods_for_component(
-            self.logging_namespace,
-            "curator",
-        )
-        if error:
-            return {"failed": True, "changed": False, "msg": error}
-        check_error = self.check_curator(curator_pods)
-
-        if check_error:
-            msg = ("The following Curator deployment issue was found:"
-                   "\n-------\n"
-                   "{}".format(check_error))
-            return {"failed": True, "changed": False, "msg": msg}
-
+        curator_pods = self.get_pods_for_component("curator")
+        self.check_curator(curator_pods)
         # TODO(lmeyer): run it all again for the ops cluster
```

Comment:
Sadly yes... these checks do nothing to examine the ops cluster. The main cluster is far more likely to fall over and affect users though so the focus is right. These little reminders are where they are because ideally we would literally just run the same high-level methods against a different set of pods/services/etc to do ops cluster checks.